gulpfile.js

```diff
@@ -1,152 +1,50 @@
-let project_folder = require("path").basename(__dirname); // <-folder to return the result
-let source_folder = "src"; // <-source folder
-
-let path = {
-  build: {
-    // paths to build
-    html: project_folder + "/",
-    css: project_folder + "/css/",
-    js: project_folder + "/js/",
-    img: project_folder + "/img/",
-  },
-  src: {
-    // paths to source files
-    html: [source_folder + "/*.html", "!" + source_folder + "/_*.html"],
-    css: source_folder + "/scss/style.scss",
-    js: source_folder + "/js/script.js",
-    img: source_folder + "/img/**/*.{jpg,png,svg,gif,ico,webp}",
-  },
-  watch: {
-    // paths to modify files
-    html: source_folder + "/**/*.html",
-    css: source_folder + "/scss/**/*.scss",
-    js: source_folder + "/js/**/*.js",
-    img: source_folder + "/img/**/*.{jpg,png,svg,gif,ico,webp}",
+const project_folder = require("path").basename(__dirname); // <-folder to return the result
```

Comment:
I think It would be more clear if you rewrite the variable name to `projectFolderName` since as a result of the execution of `require("path").basename(__dirname)` you will receive route folder name

Reply:
fix:  I agree, put the folder name in the variable